nixos/modules/services/networking/wireguard.nix

```diff
@@ -26,19 +26,28 @@ let
         type = with types; nullOr str;
         default = null;
         description = ''
-          Base64 private key generated by wg genkey.
+          Base64 private key generated by <command>wg genkey</command>.
 
           Warning: Consider using privateKeyFile instead if you do not
           want to store the key in the world-readable Nix store.
         '';
       };
 
+      generatePrivateKeyFile = mkOption {
```

Comment:
(nitpick) isn't it easier to use `mkEnableOption` here?

Reply:
I'm not sure .... would it be? :P

Reply:
I'm not entirely sure either. I guess it's mainly a matter of taste, so the `(nitpick)` prefix wasn't clear enough about that, sorry!